app/src/main/java/rkr/simplekeyboard/inputmethod/latin/inputlogic/InputLogic.java

```diff
@@ -65,6 +65,10 @@ public InputLogic(final LatinIME latinIME) {
     public void startInput() {
         mRecapitalizeStatus.disable(); // Do not perform recapitalize until the cursor is moved once
         mCurrentlyPressedHardwareKeys.clear();
+        if (mConnection.hasCursorPosition()) {
```

Comment:
Cursor position is not known at the time when this method is called. `mConnection` will probably contain last known parameters which can be totally wrong. Recapitalization is probably disabled here for a reason.